code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -1078,3 +1078,28 @@ It'll return null if the organ doesn't correspond, so include null checks when u
 	else
 		var/obj/item/organ/external/head/HD = H.get_organ("head")
 		return HD?.hair_colour
+
+
+/datum/species/water_act(mob/living/carbon/human/H, volume, temperature, source, method = REAGENT_TOUCH)
+	. = ..()
```

Comment:
add this to the existing `/datum/species/proc/water_act` instead of using . = .() here